src/containers/DashboardSettings.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default class DashboardSettings extends React.Component {
```

Comment:
If this component contains user settings, can we rename it to be `userProfile` or `userSettings`?